lib/python/satos_payload_sdk/antaris_api_gpio.py

```diff
@@ -86,8 +92,16 @@ def api_pa_pc_get_gpio_pins_number(index):
     return value
 
 def api_pa_pc_get_uart_dev():
-    value = jsfile_data[g_JSON_Key_IO_Access][g_JSON_Key_UART][g_JSON_Key_Device_Path]
-    return value
+    g_total_uart_port = jsfile_data[g_JSON_Key_IO_Access][g_JSON_Key_UART][g_JSON_Key_Device_Count]
+    uart_dev = ["NULL" , "NULL"]
```

Comment:
uart_dev size should be equal to g_total_uart_port ? If its two and g_total_uart_port is five, further code will be in trouble.

Reply:
Updated library